test/cases/HTMLOptionElement.webidl

```diff
@@ -0,0 +1,6 @@
+// Simplified from https://html.spec.whatwg.org/multipage/form-elements.html#htmloptionelement
```

Comment:
What value does having these three tests give, instead of just one of them? I can see zero-args vs. nonzero, but these three cases don't seem to differ in interesting ways.